src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -23,8 +26,9 @@
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
 
     /**
-     * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
-     * trimmed.
+     * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces
+     * will be trimmed.
+     *
```

Comment:
So we were reviewing this PR together and found line 29 and 43 to be quite funny :)

![lol](https://user-images.githubusercontent.com/38996397/95649795-f5bbb080-0b11-11eb-89d2-86572b75d1b7.jpg)

Reply:
Must be my formatter oops